serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -9,22 +9,22 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
 # BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
 # the specific language governing permissions and limitations under the License.
-ARG version=12.2.2-cudnn8-devel-ubuntu22.04
+ARG version=12.4.1-devel-ubuntu22.04
 FROM nvidia/cuda:$version
-ARG cuda_version=cu122
+ARG cuda_version=cu124
 ARG python_version=3.10
-ARG TORCH_VERSION=2.2.1
+ARG TORCH_VERSION=2.2.2
 ARG djl_version=0.28.0~SNAPSHOT
 ARG transformers_version=4.40.0
 ARG accelerate_version=0.29.3
-ARG tensorrtlibs_version=9.3.0.post12.dev1
+ARG tensorrtlibs_version=10.0.1
 ARG trtllm_toolkit_version=nightly
-ARG trtllm_version=v0.9.0
-ARG cuda_python_version=12.2.0
+ARG trtllm_version=v0.10.0_dryrun
```

Comment:
do we want to change from dryrun to standard publish?

Reply:
My bad, fixed